### PR TITLE
Enable the Test Image stage

### PR DESCRIPTION
**Pull request by jpeacock-zenoss**
Fixes BLD-230.

Test run with these changes (the actual push stages are skipped): http://platform-jenkins.zenoss.eng/job/TestBuilds/job/jpeacock/job/test_assembly/1/

### CODE REVIEW

File: Jenkins-begin.groovy [hunk 1]
```diff
@@ -132,39 +152,54 @@ node('build-zenoss-product') {
                 }
```
Comment (ealii) [inline Jenkins-begin.groovy]: Check the count of curved scopes here

Comment (jpeacock-zenoss) [reply Jenkins-begin.groovy]: I don't understand what you're asking for.

> curved scopes

Do you mean matching braces?

Comment (ealii) [reply Jenkins-begin.groovy]: yes

Comment (jpeacock-zenoss) [reply Jenkins-begin.groovy]: As far as I can tell, all the opening braces and have a matched closing brace.

Comment (ealii) [reply Jenkins-begin.groovy]: Oh, really looks good

**Pull request merged**